tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -82,15 +83,34 @@ func (r *RealDependencies) execute(cmd string, ignoreErrors bool, args ...string
 	return externalCommand.Run()
 }
 
-func (r *RealDependencies) executeXTables(cmd string, ignoreErrors bool, args ...string) error {
+func (r *RealDependencies) executeXTables(cmd string, ignoreErrors bool, args ...string) (err error) {
 	log.Infof("Running command: %s %s", cmd, strings.Join(args, " "))
 	externalCommand := exec.Command(cmd, args...)
 	stdout := &bytes.Buffer{}
 	stderr := &bytes.Buffer{}
 	externalCommand.Stdout = stdout
 	externalCommand.Stderr = stderr
 
-	err := externalCommand.Run()
+	for i := 0; i < 10; i++ {
+		stderr.Reset()
+		err = externalCommand.Run()
```

Comment:
are commands re-runnable like this? I would have thought we had to reconstruct it?

It may (or may not, not a big issue) be slightly simpler to do that anyways to avoid the resets on the buffers

Reply:
Good point! Done.